tsl/src/bgw_policy/reorder_api.c

```diff
@@ -147,6 +147,13 @@ policy_reorder_add(PG_FUNCTION_ARGS)
 				 errmsg("could not add reorder policy because \"%s\" is not a hypertable",
 						get_rel_name(ht_oid))));
 
+	if (hypertable_is_distributed(ht))
+		ereport(ERROR,
+				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+				 errmsg("add_reorder_policy not implemented for distributed hypertable"),
```

Comment:
It is good to mention the name of the distributed hypertable, e.g., as similar to the previous error.
```suggestion
				 errmsg("could not add reorder policy because \"%s\" is a distributed hypertable",
				      get_rel_name(ht_oid)),
```